forums/settings.py

```diff
@@ -120,7 +117,7 @@
 STATICFILES_FINDERS = (
     'django.contrib.staticfiles.finders.FileSystemFinder',
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
-#    'django.contrib.staticfiles.finders.DefaultStorageFinder',
+    # 'django.contrib.staticfiles.finders.DefaultStorageFinder',
```

Comment:
Please refer to Django documentation, check if this module is required here. If not, please remove